package.json

```diff
@@ -62,5 +48,10 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "node": "10.2.0"
+  },
+  "husky": {
```

Comment:
husky config that applies across the project, this replaces having `prepush` under scripts since that's being deprecated.